cloud_recording.go

```diff
@@ -0,0 +1,81 @@
+package zoom // Use this file for /recording endpoints
+
+import "fmt"
+
+const (
+	// ListAllRecordingsPath - v2 lists all recordings
+	ListAllRecordingsPath = "/users/%s/recordings"
+)
+
+// RecordingFile represents a recordings file object
+type RecordingFile struct {
+	ID             string `json:"id"`
+	MeetingID      string `json:"meeting_id"`
+	RecordingStart string `json:"recording_start"`
```

Comment:
Does this work as a `*Time`?

Reply:
According to the Zoom API, it's a `string`.

```json
.....
      "recording_files": [
        {
          "id": "string",
          "meeting_id": "string",
          "recording_start": "string",
          "recording_end": "string",
          "file_type": "string",
          "file_size": "number",
          "play_url": "string",
          "download_url": "string",
          "status": "string",
          "deleted_time": "string",
          "recording_type": "string"
.....
```

Reply:
You are correct—I was wondering if it would unmarshal correctly if you were to use `*Time` as the type. I've contacted the Zoom support team to see if there are any undocumented guarantees about data format here.

Reply:
I received confirmation from Zoom support:

> The start_time and end_time will always be a string with a format of date:time (i.e. 2019-03-21T07:30:00Z)

So for `recording_start`, `recording_end`, and `deleted_time`, please use `*Time` instead of `string`